index.js

```diff
@@ -67,26 +76,28 @@ exports.kinesisHandler = function (records, context, callback) {
 
     // POST request
     request(options, function (error, response, body) {
-      console.log('Posting...')
-      if (error || body.errors && body.errors.length) {
-        console.log(body.errors)
-        if (error) {
-          console.log('Error! ' + error)
-          callback(new Error(error))
-        } else {
-          console.log('Error! ' + body.errors)
-          callback(new Error(body.errors))
+      logger.log('info', {'message': 'Posting...'})
+      logger.log('info', {'message': 'Response: ' + JSON.stringify(response.statusCode)})
+      if (error || response.statusCode !== 200) {
+        if (response.statusCode === 401) {
+          CACHE['accessToken'] = null
         }
-      } else {
-        callback(null, 'POST Success')
+        logger.log('error', {'message': 'POST Error! ' + JSON.stringify(error)})
+        callback(new Error(error))
+        return
+      }
+      if (body.errors && body.errors.length) {
+        logger.log('info', {'message': 'Data error: ' + JSON.stringify(body.errors)})
```

Comment:
I think there might be a way to pass Winston an object and just have it handle it... and not need to do `JSON.stringify`.